frontend/src/components/pages/unloaded/Unloaded.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { CircularProgress } from '@material-ui/core';
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    loadContainer: {
+      display: "flex",
+      justifyContent: "center",
```

Comment:
This will only do horizontal centering, do we also want vertical centering on this thing?

Reply:
Yeah that would probably look better, vertical centering added.